pylint/checkers/variables.py

```diff
@@ -405,7 +405,8 @@ def _has_locals_call_after_node(stmt, scope):
     "E0601": (
         "Using variable %r before assignment",
         "used-before-assignment",
-        "Used when a local variable is accessed before its assignment.",
+        "Used when a local variable is accessed before its assignment or within "
```

Comment:
Let's merge this in the current state and use https://github.com/PyCQA/pylint/pull/5506 to perfect this description 😄